schema/artwork/index.js

```diff
@@ -320,24 +321,42 @@ const ArtworkType = new GraphQLObjectType({
       sale_message: {
         type: GraphQLString,
         args: {
-          format: {
-            type: GraphQLBoolean,
-            default: false,
-            description: 'Show a formatted response instead of the one that comes from gravity',
+          view: {
+            type: new GraphQLEnumType({
+              name: 'ArtworkViewType',
+              values: {
+                DETAIL: {
+                  value: 'detail',
+                },
+                LIST: {
```

Comment:
Right, I guess technically not, since the current UI difference is like a mutually exclusive thing, but if we're trying to encapsulate the difference via the type of artwork view it is, I )sort of_ think it's a reasonable param to have.